### PR TITLE
docs: add ecosystem section and reflection polyfill link

### PR DESCRIPTION
**Pull request by Hotell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,9 @@ $ yarn add injection-js
 > For ES5 `Class` syntax and TypeScript you need a polyfill for the [Reflect API](http://www.ecma-international.org/ecma-262/6.0/#sec-reflection).
 > You can use:
 >
+> - [reflection](https://www.npmjs.com/package/@abraham/reflection) 👉 only 3kb (recommended)
```
Comment (mgechev) [inline README.md]: I saw this in the mailing list yesterday. Maybe too early to be recommended? :-)

Comment (Hotell) [reply README.md]: I'm already using it so all good 😎 . I've talked with Abraham about but via GitHub issues as well. But no biggie I can remove word "recommended" 😉

Comment (mgechev) [reply README.md]: Yep, let's remove it for now. I think 3K is enough of a teaser already :)

**Comment by Hotell** [comment]
updated 🤙

**Review by mgechev — APPROVED** [review]

**Pull request merged**